Functions/Gherkin.ps1

```diff
@@ -787,3 +787,150 @@ function ConvertTo-HashTableArray {
     }
 }
 
+function Import-FeatureFile {
+    param(
+        # Path to Search
+        [Parameter(ValueFromPipeline = $true)]
+        [string[]]
+        $Path = $pwd
+    )
+    
+    begin {
+        $parser = New-Object Gherkin.Parser
+    }
+
+    process {
+        $item = Get-Item -Path $Path
+
+        if ($item.psisContainer) {
+            foreach ($file in $path | Get-ChildItem -Recurse -Filter *.feature) {
+                try {
+                    $fixtureFilename = "{0}.Steps.ps1" -f $file.BaseName
+                    $fullName = Join-Path -Path $file.Directory -ChildPath $fixtureFilename
+                    $parsed = $parser.parse($file.FullName)
+
+                    if (Test-Path -Path $fullName) {
+                        Update-TestStubs -stubFile $fullName -parsedInput $parsed
+                    }
+                    else {
+                        $stubFile = New-Item -Path $fullName -ItemType File
+                        
+                        Add-Stubs -stubFile $stubFile -parsedInput $parsed   
+                    }
+                }
+                catch {
+                    throw $_
+                }
+            }   
+        }
+        else {
```

Comment:
You're duplicating code here. You need:
* One function that parses a file (it already exists, it's called `Import-GherkinFeature` and you're re-doing it here)
* Another function that enumerates all the feature files and calls the `Import-GherkinFeature` function for each.

If we need to, we can _refactor_ `Import-GherkinFeature` to return an object or a hashtable instead of an array so that you can re-use it here, but we don't want to duplicate the whole parser/parse/enumerate thing...